docs/pages/application-access/guides/azure.mdx

```diff
@@ -0,0 +1,604 @@
+**User-assigned managed identity**. Choose the `teleport-azure` identity you
+created earlier. 
+
+![Select managed
+identities](../../../img/application-access/azure/select-managed-identities.png)
+
+Click **Select**, then **Review + assign**.
+
+Verify that your **Role** is "Reader", the **Scope** matches your chosen
+resource group, and the **Members** field includes the `teleport-azure` managed
+identity you created earlier.
+
+Click **Review + assign** again.
+
+### Attach an identity to your Azure VM
+
+Now that you have created a managed identity and assigned it a role, attach the
+identity to your Azure VM so the Teleport Application Service can assume the
+identity in order to proxy Azure CLI traffic.
+
+In the [Virtual machines
+view](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Compute%2FVirtualMachines)
+of Azure Portal, click on the name of the VM you are using to host the Teleport
+Application Service. 
+
+On the right side panel, click the **Identity** tab, then within the
+**Identity** view, click the **User assigned** tab. Click **+Add**, then select
+the `teleport-azure` identity. Click **Add**.
+
+![Add an identity to a
+VM](../../../img/application-access/azure/vm-identity.png)
+
+Navigate back to **Identity** tab in the page for your Azure VM. You should see
+the new identity listed in the **User assigned** sub-tab:
+
+![Verifying that you added the
+identity](../../../img/application-access/azure/verify-id.png)
+
+## Step 2/4. Deploy the Teleport Application Service
+
+In this step, you will run the Teleport Application Service on the Azure VM you
+assigned the `teleport-azure` identity to. 
+
+### Get a join token
+
+Establish trust between your Teleport cluster and your new Application Service
+instance by creating a join token:
+
+```code
+$ tctl nodes add --roles=app
+The invite token: (=presets.tokens.first=)
+This token will expire in 30 minutes.
+Run this on the new node to join the cluster:
+> teleport start \
+   --roles=app \
+   --token=(=presets.tokens.first=) \
+   --ca-pin=(=presets.ca_pin=) \
+   --auth-server=192.0.2.255:3025
+Please note:
+  - This invitation token will expire in 30 minutes
+  - 192.0.2.255:3025 must be reachable from the new node
+```
+
+On the host where you will install the Teleport Application Service, create a
+file called `/tmp/token` that consists only of your token:
+
+```code
+$ echo <Var name="join-token" /> | sudo tee /tmp/token
+```
+
+### Install the Teleport Application Service
+
+Run the following commands on the host where you will install the Teleport
+Application Service:
+
+<Tabs dropdownView dropdownCaption="Teleport Edition">
+<TabItem label="Self-Hosted" scope={["oss", "enterprise"]}>
+
+(!docs/pages/includes/install-linux.mdx!)
+
+</TabItem>
+<TabItem label="Teleport Cloud" scope="cloud">
+
+(!docs/pages/includes/install-agent-cloud.mdx service="the Teleport Kubernetes Service"!)
```

Comment:
Kubernetes service isn't correct for this context, though I'm also not seeing this var being used in the render.

Reply:
That was a lazy copy/paste oversight on my part. That partial had a parameter at one point, but I removed it because the docs linter was (at the time, but not now) erroring on partials with default param values.